### PR TITLE
Fix GNOME fallback/flashback on trusty+,wheezy+,kali+

### PR DESCRIPTION
**Pull request by smibarber**
Addresses feedback in #2429.

### CODE REVIEW

File: targets/xorg [hunk 1]
```diff
@@ -205,8 +205,7 @@ else
 fi
 
 # Fix launching X11 from inside crosh (user doesn't own a TTY)
-sed -i 's/allowed_users=.*/allowed_users=anybody/' \
-    '/etc/X11/Xwrapper.config' || true
+echo 'allowed_users=anybody' > '/etc/X11/Xwrapper.config'
```
Comment (drinkcat) [inline targets/xorg]: Are we sure that `/etc/X11/Xwrapper.config` never contains anything else of interest?

If not, we might want to write a small awk script instead to modify/add the line as required.

Comment (dnschneid) [inline targets/xorg]: The only other configuration items, it seems, are needs_root_rights and nice_value, which aren't even documented.  So no, there's nothing else.

**Comment by dnschneid** [comment]
Seems to be getting `[e] Error executing command as another user: Not authorized` on x86 sid

**Comment by dnschneid** [comment]
Tests seem to be passing for the most part...good job!

**Comment by onesneakymofo** [comment]
Just want to thank you guys for releasing this update.  I bricked my installation about two weeks ago, and this fixed it.

**Pull request merged**